### ISSUE TITLE
Clicking Add failure instance doesn't work in Keyboard assessment

### ISSUE THREAD

**Issue by AhmedAbdoOrtiga** [microsoft/accessibility-insights-web#1112]
## Describe the bug
Clicking Add failure instance doesn't work in Keyboard assessment, Keyboard Navigation step if the Assessment Page is on the **Right Side of the Screen**

**To Reproduce**

1. Go to any web page.
2. Switch to Keyboard assessment, Keyboard Navigation step.
3. Click window + right arrow so the assessment page would be in the right half of the screen.
4. Click fail.
5. Then click "Add a failure instance".
6. Most of the times it doesn't work or you need to click more than once.

## Expected behavior

"Add a failure instance" panel should show up

## Screenshots
![image](https://user-images.githubusercontent.com/45858632/63393093-6207ad80-c36e-11e9-9440-a8eac171c30a.png)

## Extension (please complete the following information)

- OS: [Windows]
- Version [Insider 2.8.1]
- Browser Version [Chrome 76.0.3809.100]
- Target Page [any page eg: www.microsoft.com]

**Comment by karanbirsingh** [comment]
It seems that clicking on Add Failure Instance can cause some of the text to wrap, which can jerk the button's position a bit down. May have been introduced with recent change to failure instance "Comment:" UI

**Comment by dbjorge** [comment]
This was introduced by #1098. It repros on Chromium 75 and 76, but not on 77 or 78.

We're reverting #1098 (in #1119) to fix this bug for the short term, and will re-apply #1098 once Chromium 77 hits stable and this bug will no longer apply.
